app/helper/rtForm.php

```diff
@@ -569,6 +572,60 @@ protected function generate_textbox( $attributes ) {
 			}
 		}
 
+		/**
+		 * Generate rtmedia html inputfile in admin options.
+		 *
+		 * @access protected
+		 *
+		 * @param $attributes
+		 *
+		 * @return string
+		 * @throws rtFormInvalidArgumentsException
+		 */
+		protected function generate_inputfile( $attributes ) {
+
+			$element = 'rtFile';
+			if ( is_array( $attributes ) ) {
+
+				/* Starting the input tag */
+				$html = '<input type="file" ';
+
+				/* Generating attributes */
+				$html .= $this->processAttributes( $element, $attributes );
+
+				/* Ending the tag */
+				$html .= ' /><input type="hidden" name="rtmedia-options[' . $attributes['name'] . '_hid]" value="' . $attributes['value'] . '" />';
+				if ( ! empty( $attributes['value'] ) ) {
+					$img_src  = wp_get_attachment_image_src( $attributes['value'], 'thumbnail' );
+					$img_path = get_attached_file( $attributes['value'] );
+
+					if ( file_exists( $img_path ) && ! empty( $img_src[0] ) ) {
+						$html .= '<span class="rtm-file-preview">';
+						$html .= '<img src="' . $img_src[0] . '" width="100">';
```

Comment:
use proper escaping always process string with a variable in `sprintf`
eg: 
``` $html .= '<img src="' . $img_src[0] . '" width="100">'; ```
to
``` $html .= sprintf( '<img src="%s" width="100">, esc_url( $img_src[0] ) ); ```